### PR TITLE
Undo/Redo and Editing Notes Scenarios are added, notes from previous pull request are applied

### PR DESCRIPTION
**Pull request by GitRepoForHanna**
notes from pull request are applied

### CODE REVIEW

File: src/main/java/pages/BottomMenuPage.java [hunk 1]
```diff
@@ -34,32 +34,35 @@ public BottomMenuPage() {
     }
 
     public boolean isDisplayed() {
-        if (AndroidDriverSingletone.getSingletoneInstance().getDriverInstance().findElements(By.id(BOTTOM_MENU_ID)).size() == 0)
-            return false;
-        return bottomMenu.isDisplayed();
+        return (AndroidDriverSingletone.getSingletoneInstance().getDriverInstance().findElements(By.id(BOTTOM_MENU_ID)).size() != 0) && bottomMenu.isDisplayed();
```
Comment (MihailDamasheuski) [inline src/main/java/pages/BottomMenuPage.java:37]: Please implement new methods in BasePage:
protected List<WebElement> findElementsById(String elementId)
protected WebElement findElementById(String elementId)
protected List<WebElement> findElementsByXpath(String elementXpath)
protected WebElement findElementByXpath(String elementXpath)

- [ ] Implement methods

File: src/main/java/steps/NotePageSteps.java [hunk 2]
```diff
@@ -48,16 +47,28 @@ public void setNoteData(Note note) {
...
 
+    public void clearTitle() {
+        new NotePage().clearTitle();
+    }
+
     public void setTitle(Note note) {
+        new NotePage().setTitle(note.getName());
+    }
+
+    public void updateTitle(Note note) {
```
Comment (MihailDamasheuski) [inline src/main/java/steps/NotePageSteps.java:58]: This method is not related to update, it's more about change title.
I suggest you implement one more method for update: 
NotePage notePage = new NotePage();
        String currentTitle = notePage.getNotTitle();
        notePage.setTitle(updateTitle(currentTitle, note.getName()));

File: src/test/java/EditingNotes.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+public class EditingNotes extends BaseTest{
+
+    @Test(dataProviderClass = NoteDataProvider.class, dataProvider = "EditNoteTitle")
+    public void editNoteTitle(Note editedNote, Note desiredNote) {
+        SoftAssert softAssert = new SoftAssert();
+        String oldTitle = editedNote.getName();
+        String newTitle = desiredNote.getName();
+        baseEmulatorPageSteps.preparationActivities();
+        softAssert.assertEquals(0, homePageSteps.getNotesCount(), "There are no notes on Hame page");
```
Comment (MihailDamasheuski) [inline src/test/java/EditingNotes.java:16]: Sequence of parameter in assertions should be: actial, expected

- [x] Update perams sequence

**Review by MihailDamasheuski — APPROVED** [review]

File: src/test/java/EditingNotes.java (hunk 3 above)
Comment (GitRepoForHanna) [reply src/test/java/EditingNotes.java:16]: done
